IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py

```diff
@@ -713,6 +713,19 @@
     Z0 = cms.double(0.645)
 )
 
+# From fit to 5 TeV PbPb data
```

Comment:
@ColeLindsey it is considered good practice to document in this file how the numbers reported in the configuration have been obtained by breaking down the contribution from the Beam Spot and the pixel barycenter positions.
You might refer to this:
https://github.com/cms-sw/cmssw/blob/f084ec018ce6e9998d89201674f4a9fe37465fea/IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py#L597-L612
as a good example.